### PR TITLE
APP-5488: add IDs to rc cards

### PR DESCRIPTION
**Pull request by ethanlook**
This adds IDs to each RC card. By doing this, all cards (new and old) will have fragments we can link to, so we can show all resources in the sidebar in App.

### CODE REVIEW

File: web/frontend/src/lib/components/collapse.svelte [hunk 1]
```diff
@@ -36,7 +36,7 @@ onMount(() => {
 });
 </script>
 
-<div class="relative w-full">
+<div class="relative w-full" id={title}>
```
Comment (DTCurrie) [inline web/frontend/src/lib/components/collapse.svelte:39]: Any cases where a `title` may be a non-ideal `id` value, like spaces or anything?

**Review by DTCurrie — APPROVED** [review]
✅

File: web/frontend/src/lib/components/collapse.svelte (hunk 1 above)
Comment (ethanlook) [reply web/frontend/src/lib/components/collapse.svelte:39]: "Operations & sessions" is the only one. I can `encodeURIComponent`

File: web/frontend/src/lib/components/collapse.svelte [hunk 2]
```diff
@@ -36,7 +36,10 @@ onMount(() => {
 });
 </script>
 
-<div class="relative w-full" id={title}>
+<div
+  class="relative w-full"
+  id={encodeURIComponent(title)}
```
Comment (zaporter-work) [inline web/frontend/src/lib/components/collapse.svelte:41]: Any chance that this element id is used elsewhere in app? IIRC, it is bad to have multiple elements with the same ID

**Review by zaporter-work — APPROVED** [review]

File: web/frontend/src/lib/components/collapse.svelte (hunk 2 above)
Comment (ethanlook) [reply web/frontend/src/lib/components/collapse.svelte:41]: You are correct, but there shouldn't be uses of the same ID on the control page. On the config page there are conflicts, but this component isn't rendered there.

**Pull request merged**